modules/sdk/gradle-plugins-whip/README.markdown

```diff
@@ -37,8 +37,9 @@ repositories {
 }
 ```
 
-By default, Whip is automatically applied to all the tasks of type [`Test`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html).
-If a task has Whip applied and it is [enabled](#enabled), then Whip is configured as a Java Agent.
+By default, Whip is automatically applied to all tasks of type [`Test`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html).
+If a task has Whip applied and is [enabled](#enabled), then Whip is configured
+as a Java Agent.
```

Comment:
@codyhoag is right, the task must have Whip applied, and the `task.whip.enabled` property must be `true`.